src/monopoly/monopoly_player.js

```diff
@@ -75,13 +71,15 @@ MonopolyPlayer.prototype.owesRent = function(property) {
 };
 
 MonopolyPlayer.prototype.owns = function(property) {
-  return property.owner === this;
+  return property.owner && property.owner.id === this.id;
```

Comment:
I need to refocus for a bit so it can probably wait until everything is fixed.

> On Feb 23, 2016, at 11:07 PM, andrewjmeier <notifications@github.com> wrote:
> 
> In src/monopoly/monopoly_player.js <https://github.com/andrewjmeier/TableTop.JS/pull/91#discussion_r53891579>:
> 
> > @@ -75,13 +71,15 @@ MonopolyPlayer.prototype.owesRent = function(property) {
> >  };
> >  
> >  MonopolyPlayer.prototype.owns = function(property) {
> > -  return property.owner === this;
> > +  return property.owner && property.owner.id === this.id;
> Do you want to merge it or wait for me to fix those few things? I'm going to do some grading for the next hour or so first
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/andrewjmeier/TableTop.JS/pull/91/files#r53891579>.
>